models/basgra/R/write.config.BASGRA.R

```diff
@@ -372,7 +594,7 @@ write.config.BASGRA <- function(defaults, trait.values, settings, run.id, IC = N
     }
     
     # Initial mineral N
-    nmin0 <- try(ncdf4::ncvar_get(IC.nc, "soil_nitrogen_content"), silent = TRUE)
+    nmin0 <- try(ncdf4::ncvar_get(IC.nc, "soil_inorganic_nitrogen_content"), silent = TRUE)
     if (!is.na(nmin0) && is.numeric(nmin0)) {
       run_params[which(names(run_params) == "NMIN0")] <- PEcAn.utils::ud_convert(nmin0, "kg", "g")
     }
```

Comment:
```suggestion
    }
    
        # Initial organic N
    nsom0 <- try(ncdf4::ncvar_get(IC.nc, "soil_organic_nitrogen_content"), silent = TRUE)
    if (!is.na(nsom0) && is.numeric(nsom0)) {
      run_params[which(names(run_params) == "NSOM")] <- PEcAn.utils::ud_convert(nsom0, "kg", "g")
    }
```
maybe let's add this too